src/intro/tooling.md

```diff
@@ -15,15 +15,13 @@ tested.
 - OpenOCD >=0.8. Tested versions: v0.9.0 and v0.10.0
 - GDB with ARM support. Version 7.12 or newer highly recommended. Tested
   versions: 7.10, 7.11, 7.12 and 8.1
-- [OPTIONAL] `git` OR
-  [`cargo-generate`](https://github.com/ashleygwilliams/cargo-generate). If you
-  have neither installed then don't worry about installing either.
+- [`cargo-generate`](https://github.com/ashleygwilliams/cargo-generate) and `git`.
```

Comment:
This seems fine to me

Reply:
Actually, this should be "(..) or `git`". You don't need *both* tools; one of them suffices.

Reply:
Later on in the book `cargo generate --git` command is used. We can put `or` here when comes with it's own git api and doesn't rely on standalone git. I assumed it relies on standalone git.

Reply:
From this wording i conclude that standalone git is not required https://github.com/rust-lang/git2-rs#version-of-libgit2 (dependency of cargo generate)

Reply:
`cargo-generate` doesn't shell out to `git`; it uses libgit.